tools/docker/tests/test_flows.sh

```diff
@@ -35,7 +35,7 @@ send_bml_command() {
 
 send_CAPI_command() {
     # send a CAPI command to a container.
-    ${topdir}/prplMesh/tools/docker/tests/send_CAPI_command.py $(container_ip "$1") $(container_CAPI_port "$1") "$2"
+    ${rootdir}/prplMesh/tools/docker/tests/send_CAPI_command.py $(container_ip "$1") $(container_CAPI_port "$1") "$2"
```

Comment:
How can this work? shouldn't this be `${rootdir}/tools/docker/tests/send_CAPI_command.py $(container_ip "$1") $(container_CAPI_port "$1") "$2"` ?

Reply:
This commit was generated using the command in the commit message. The next commit fixes this manually.
It would have been possible to include this particular change in the first (generated) commit, but other manual fixes are needed, so the 2 commits _have_ to be applied together anyway.

Reply:
I now see that e74d848c7e71dc3bba8275f5a3db2c1436885cc3 fixes that but IMHO it should be squashed to the commit before that, otherwise this commit "breaks" `build.sh`, doesn't it?

Reply:
2c435b4a indeed does break things, it has to be applied with the next one (e74d848c).
I remember @arnout mentioned he prefers to have "generated" commits separate from their manual fixes, but I don't know if it applies in this case.

Reply:
I asked @arnout, the reason to have them split was to make it easier to review.
Since you both already did, I'll squash them now.